src/app/features/ledger/flows/stacks-message-signing/ledger-stacks-sign-msg-container.tsx

```diff
@@ -140,27 +140,20 @@ function LedgerSignStacksMsg({ account, unsignedMessage }: LedgerSignMsgProps) {
     }
   }
 
-  const allowUserToGoBack = get(location.state, 'goBack');
```

Comment:
I removed this `allowUserToGoBack` as I can't find where `'goBack'` is set in the state and I don't think it's doing anything